core/trino-main/src/main/java/io/trino/operator/BigintGroupByHash.java

```diff
@@ -22,6 +22,7 @@
 import io.trino.spi.block.Block;
 import io.trino.spi.block.BlockBuilder;
 import io.trino.spi.block.DictionaryBlock;
+import io.trino.spi.block.LongArrayBlock;
 import io.trino.spi.block.RunLengthEncodedBlock;
```

Comment:
Do we need all the benchmarking cases since you just benchmarked subset?

Reply:
It benchmarks not only bigints.
But if you like I can remove them from here and put them in PRs related to multi channel aggregation